src/main.rs

```diff
@@ -36,6 +36,13 @@ fn main() {
         )
         .required(true);
 
+    let cmd_duration_arg = Arg::with_name("elapsed_time")
```

Comment:
I think we should stick to `cmd_duration` here too.
As I understand it, this is a measure of the duration of the last command's execution time, not the time elapsed since the start of the last command; or am I misunderstanding how it is implemented?

Reply:
Yes, that's correct. I was just mentally doing a manual string replace and missed this part.